### ISSUE TITLE
Regarding the loss of nan when training grayscale images

### ISSUE THREAD

**Issue by damaha-9527** [fengyurenpingsheng/WeConvene#1]
Thank you very much for your work, but when I trained a grayscale image with only one channel dimension using the code you provided, at about 30 rounds, all the losses became nan. What could be the possible reason

**Comment by fengyurenpingsheng** [comment]
The framework is suitable for RGB images, may be for your task you can reduce the learning rate from e-4 to e-5.
